MapboxNavigation/NavigationViewController.swift

```diff
@@ -147,6 +147,18 @@ open class NavigationViewController: UIViewController, NavigationStatusPresenter
      */
     public var shouldManageApplicationIdleTimer = true
     
+    /**
+     Enables highlighting of the destination building when arriving.
+     */
+    public var highlightDestinationBuildings: Bool = false
+    
+    /**
+     When destination building highlighting is enabled, this option controls whether the building will be presented in 3d or 2d.
```

Comment:
I don't think that `highlightsBuildingsIn3D` is ignored when `highlightsDestinationBuildings` is enabled, it just doesn't have any effect since `highlightsDestinationBuildings` is set to `false` by default and thus buildings are not extruded.

Reply:
Yes, that’s what I mean by it being ignored. Essentially we have a three-way setting, or a setting that depends on another setting. We should avoid such situations in Swift.

Reply:
Change two boolean variables to `DestinationBuildingHighlightType`. What do you think about such public API/naming?